### PR TITLE
Code cast detail screen

### PR DESCRIPTION
**Pull request by nvquangth**
Function: feature cast detail
Redmine: https://edu-redmine.framgia.vn/issues/1339

### CODE REVIEW

**Comment by nvquangth** [comment]
ready

File: app/src/main/java/com/quangnv/moviedb/screen/moviedetail/MovieDetailActivity.java [hunk 1]
```diff
@@ -83,15 +93,22 @@ public void onFavoriteClick(Movie movie) {
...
     }
-    private void showReadMoreDescription(Movie movie) {
-        FragmentTransaction transaction = getFragmentManager().beginTransaction();
-        Fragment prev = getFragmentManager().findFragmentByTag(TAG_DIALOG_FRAGMENT_DES);
-        if (prev != null) {
-            transaction.remove(prev);
+
+    private void showReadMoreDescription(Movie movie){
+        FragmentTransaction transaction=getFragmentManager().beginTransaction();
+        Fragment prev=getFragmentManager().findFragmentByTag(TAG_DIALOG_FRAGMENT_DES);
```
Comment (tuannguyen38) [inline app/src/main/java/com/quangnv/moviedb/screen/moviedetail/MovieDetailActivity.java]: format code

File: app/src/main/java/com/quangnv/moviedb/screen/cast/CastActivity.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case android.R.id.home:
+                onBackPressed();
+                break;
+            case R.id.navigation_search:
+                startActivity(getSearchIntent(this));
+                break;
```
Comment (tuannguyen38) [inline app/src/main/java/com/quangnv/moviedb/screen/cast/CastActivity.java]: add default case.

File: app/src/main/java/com/quangnv/moviedb/screen/moviedetail/MovieDetailActivity.java [hunk 3]
```diff
@@ -83,15 +93,22 @@ public void onFavoriteClick(Movie movie) {
...
         mBinding.setViewModel(mViewModel);
     }
-    private void showReadMoreDescription(Movie movie) {
-        FragmentTransaction transaction = getFragmentManager().beginTransaction();
-        Fragment prev = getFragmentManager().findFragmentByTag(TAG_DIALOG_FRAGMENT_DES);
-        if (prev != null) {
-            transaction.remove(prev);
+
+    private void showReadMoreDescription(Movie movie){
+        FragmentTransaction transaction=getFragmentManager().beginTransaction();
```
Comment (tuannguyen38) [inline app/src/main/java/com/quangnv/moviedb/screen/moviedetail/MovieDetailActivity.java]: formatcode

**Comment by nvquangth** [comment]
ready

**Pull request merged**